README.md

```diff
@@ -967,7 +967,9 @@ not persisted between runs.
 
 One or more caches of parsed Puppet modules and manifests can be created to skip the use
 of kafo_parsers at runtime. This is useful when kafo_parsers doesn't support the
-version of Puppet in use, and may also provide a small performance benefit.
+version of Puppet in use, and may also provide a small performance benefit. When multiple
+cache files are used, they are being loaded in order they are specified in the config
+file. If their files overlap, the later overrides earlier.
```

Comment:
@ares yeah, that's a correct description of how it works. 

Here I have two fake extremely simple parser caches.

```yaml
# fake_foreman.yaml
---
  !ruby/sym version: 1
  !ruby/sym files:
    foo: foo
    bar: bar
    baz: baz
```

```yaml
# fake_katello.yaml
---
  !ruby/sym version: 1
  !ruby/sym files:
    foo: different foo
    fizz: fizz
    buzz: buzz
```

You can see here how they are merged. Note that they both share a `"foo"` key, and the second key from `fake_katello.yaml` is what's accepted into the final cache because it comes second in the list of cache files.

```
Kafo::ParserCacheReader.new_from_file(['/home/vagrant/parser-cache-scratch/fake_foreman.yaml','/home/vagrant/parser-cache-scratch/fake_katello.yaml'])
=> #<Kafo::ParserCacheReader:0x00000001bbde78
 @cache={:version=>1, :files=>{"foo"=>"different foo", "bar"=>"bar", "baz"=>"baz", "fizz"=>"fizz", "buzz"=>"buzz"}}>
```